independent-publisher-connectors/Fuxsy-AutodeskFusionManage/readme.md

```diff
@@ -0,0 +1,123 @@
+### Fuxsy-AutodeskFusionManage
```

Comment:
Hello @rmnkvrk, 

Kindly follow the below link and update the readme.md file with all the required sections along with headings.

Reply:
Thanks for reminder for some reason we havent updated you last time latest version of Readme,
Please check that now.

thakn you for your help.